ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -26,6 +26,13 @@ def fill(dims, value, name="full"):
     return ivy.full(dims, value)
 
 
+def linear(x, name="linear"):
+        return ivy.linear(x)
```

Comment:
the `ivy.linear` function is a bit different from `tf.linear`, according to the docs - https://www.tensorflow.org/api_docs/python/tf/keras/activations/linear 

inputs are returned unmodified, I think you can change it so that it `return x`